src/Http/Requests/QuestionnaireFrontReadRequest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace EscolaLms\Questionnaire\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class QuestionnaireFrontReadRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    public function rules(): array
+    {
+        return [];
+    }
+
+    public function getParamId()
+    {
+        return $this->route('id');
```

Comment:
You can add validation to route params by using `prepareForValidation` method and calling `$this->merge(['id' => $this->route('id')])`, then you can use rules() for validating `id` field as if it was normal input field

I'm still sad we don't use automatic model binding :(

Reply:
done